app/views/providers/merits_task_lists/show.html.erb

```diff
@@ -14,39 +13,48 @@
       </p>
     <% end %>
 
-    <h3 class="govuk-heading-m">1. <%= t(".application_section_heading") %></h3>
+    <ol class="govuk-list govuk-list--number govuk-heading-m">
```

Comment:
nit: adding the class`govuk-heading-m` to the `ol` seems wasteful  as the style would get applied to all content, until the another class/id overrides it.

not a blocker.